modules/govuk/manifests/node/s_backend_lb.pp

```diff
@@ -108,6 +108,8 @@
   }
 
   nginx::config::vhost::proxy { "docs.${app_domain}" :
-    to => ['govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com'],
+    to        => ['govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com'],
+    protected => false,
+    http_host => 'govuk-developer-documentation-production.s3-website-eu-west-1.amazonaws.com',
```

Comment:
Thanks for the clarifying comment. Without having any real insight in to the potential usage of the buckets I don't know if it's worth having per environment setups to enable future testing of changes. I'm happy to follow your lead.